sources/api/thar-be-settings/src/main.rs

```diff
@@ -179,13 +178,27 @@ async fn run() -> Result<(), Box<dyn std::error::Error>> {
                 "Requesting affected services for settings: {:?}",
                 &changed_settings
             );
-            let services =
-                service::get_affected_services(&args.socket_path, Some(changed_settings)).await?;
-            trace!("Found services: {:?}", services);
-            if services.is_empty() {
+
+            let service_settings_map = service::get_affected_service_setting_map(
```

Comment:
This should not take an option and should not return an option (since the only invocation of this function always give Some(x)).